conf/config_canvecaux.yaml

```diff
@@ -63,7 +62,18 @@ training:
 
 inference:
   img_dir_or_csv_file: ./data/trn_val_tst_kingston.csv  # https://drive.google.com/file/d/1uNizOAToa-R_sik0DvBqDUVwjqYdOALJ
-  working_folder: ./data/output
-  state_dict_path: ./data/output/checkpoint.pth.tar
+  working_folder:
+  state_dict_path: ./data/kingston_wv2_40cm/images/samples256_overlap78_min-annot10/model/config_canvecaux/checkpoint.pth.tar
```

Comment:
- samples folder are named taking into account sample_size, overlap and min_annot_perc
- state_dict are now saved under samples directory to keep better traces of origin of model.